metabase/metabase
# Respect chart orders in static viz

Epic: #24759
[Notion doc](https://www.notion.so/metabase/Viz-Pod-Rest-of-the-Cycle-Outlook-c0456a67b5754647937bdc1def78e04c#211dba1718754379a195916499158369)

## Changes
- Respect order settings in combo chart `graph.series_order`
- Respect order settings in funnel chart `funnel.rows`

## How to test
1. Create one or more questions with 2 dimensions, select bar, or area, or line as their visualization and reorder the breakout columns, then toggle their visibility as desired.
1. Create one or more funnel charts questions, reorder the step and toggle their visibility as desired.
1. Add those questions to a dashboard
1. Send a test subscription for that dashboard and see the result

#### App
- ![image](https://user-images.githubusercontent.com/1937582/204808753-de3d1c3e-6a31-40ca-8cc1-0a35e3fb8b14.png)
- ![image](https://user-images.githubusercontent.com/1937582/204808698-c0c274fc-5dc6-44d5-9907-c21018dd86e1.png)


#### After
- ![image](https://user-images.githubusercontent.com/1937582/204808609-3ccb80f8-3604-49d0-a714-5acc2537fa29.png)
- ![image](https://user-images.githubusercontent.com/1937582/204808485-69451803-6184-410b-8390-efd4816a5c7d.png)

#### Before
- ![image](https://user-images.githubusercontent.com/1937582/204808973-83a5b83f-e726-48fe-9552-34a8090ce2c1.png)
- ![image](https://user-images.githubusercontent.com/1937582/204809269-b3936fd3-cf70-4b53-93d5-a4006e613501.png)
